keras_nlp/src/bounding_box/converters_test.py

```diff
@@ -0,0 +1,240 @@
+# Copyright 2024 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import itertools
+
+import numpy as np
+import pytest
+from absl.testing import parameterized
+from keras import backend
+
+from keras_nlp.src import bounding_box
+from keras_nlp.src.tests.test_case import TestCase
+
+try:
```

Comment:
no need to conditional tf import in tests

Reply:
Done